src/diffusers/pipelines/vq_diffusion/pipeline_vq_diffusion.py

```diff
@@ -64,6 +65,7 @@ def __init__(
         tokenizer: CLIPTokenizer,
         transformer: Transformer2DModel,
         scheduler: VQDiffusionScheduler,
+        learned_classifier_free_sampling_embeddings: LearnedClassifierFreeSamplingEmbeddings,
```

Comment:
That's definitely the right way to do it - it's quite specific to vq-diffusion IMO though, so will move it here :-)